source/localizable/_code-of-conduct.en.md

```diff
@@ -70,4 +70,4 @@ We expect all community participants (contributors, paid or otherwise; sponsors;
 License and attribution
 -----------------------
 
-Berlin Code of Conduct is distributed under a [Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/) license. It is based on the [pdx.rb code of conduct](http://pdxruby.org/codeofconduct).
+Berlin Code of Conduct is distributed under a [Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/deed.en) license. It is based on the [pdx.rb code of conduct](http://pdxruby.org/codeofconduct).
```

Comment:
I can’t see any difference in the content of the old and updated URL. That’s not a problem. I’m just curious why.

Reply:
Good catch. Yes, you are right. Content is the same since english seems to be the default. It's just to be aligned with the other locale links. :)